python/test/unit/language/test_core.py

```diff
@@ -5917,7 +5917,7 @@ def test_side_effectful_reduction(device):
     if device != "cuda":
         pytest.skip()
 
-    @triton.jit(debug=True)
+    @triton.jit
```

Comment:
I think the `debug=True` needs to be added as a kwarg to the invocation of the triton kernel. Previously I wasn't seeing any asserts in the ttgir

Reply:
Thanks for spotting this, I've opened #5037 to fix it